test/uk/gov/hmrc/pillar2stubs/controllers/UKTRSubmitControllerSpec.scala

```diff
@@ -18,7 +18,8 @@ package uk.gov.hmrc.pillar2stubs.controllers
 
 import org.scalatest.OptionValues
 import org.scalatest.freespec.AnyFreeSpec
-import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import org.scalatest.matchers.must.Matchers.mustBe
+import org.scalatest.matchers.must.Matchers.mustEqual
 import org.scalatest.matchers.should.Matchers
```

Comment:
This one's a bit closer, but `must` is more frequently used here. Could we please convert the assertions here to just use `must` as above?
```suggestion
import org.scalatest.matchers.must.Matchers
```